dataline-webapp/src/pages/SourcesPage/pages/AllSourcesPage/components/LastSyncCell.tsx

```diff
@@ -16,20 +16,20 @@ const Content = styled.div<{ enabled?: boolean }>`
 `;
 
 type IProps = {
-  value: number;
```

Comment:
What's the reason behind this change? Don't think this renaming is required.

Reply:
In case we want to bring more meaning to this value field for Cell components - I think then we should get rid of value part and use just `seconds` + similar changes should be applied in other cells too.

Reply:
might not be necessary for all the other cells but when we manipulate time as long or integer, we need to have some sort of signal that it is second or millisecond.
I'll rename this one to `timeInSecond`